static/js/impala/persona_creation.js

```diff
@@ -217,21 +217,21 @@
                 });
             };
 
-        $d.delegate('.reset', 'click', _pd(function() {
+        $d.on('click', '.reset', _pd(function() {
             var $this = $(this),
                 $p = $this.closest('.row');
             $p.find('input[type="hidden"]').val('');
             $p.find('input[type=file], .note').show();
-            $p.find('.preview').removeAttr('src').removeClass('loaded');
```

Comment:
okay.  I wasn't sure if removing it would change the property though (the migration guide specifically talks about boolean values, but also talks generically about DOM objects).  I'll have a play with the persona tools.